src/vault.cairo

```diff
@@ -0,0 +1,121 @@
+#[starknet::interface]
+trait IVaultContract<TContractState> {
+    fn store_liquidity(ref self: TContractState, amount: u256);
```

Comment:
Please, for amounts use the `TokenAmount` [alias](https://github.com/djeck1432/spotnet/blob/main/src/types.cairo#L5) type as it'll give more semantic context